out_of_tree_boards/boards/arm/npcx4m8f_ptl/npcx4m8f_ptl.dts

```diff
@@ -21,8 +21,6 @@
 	aliases {
 		i2c0 = &i2c2_0;
 		i2c1 = &i2c0_0;
-		ssd-i3c-bus = &i3c2;
```

Comment:
@Asthachavda  you should squash this commit with https://github.com/intel/ecfw-zephyr/pull/78/commits/86dbc4a13553195f20b719e4d28436789558ee54